sdk/identity/identity/TOKEN_CACHING.md

```diff
@@ -37,6 +41,12 @@ With persistent disk token caching enabled, the library first determines if a va
 - Ensures the app can continue to function during an Azure AD outage or disruption.
 - Avoids having to prompt users to authenticate each time the process is restarted.
 
+#### Implementation note
```

Comment:
Remove this section, as discussed in the team chat.

Reply:
done